ui/hooks/usePolling.ts

```diff
@@ -20,6 +24,7 @@ const usePolling = (usePollingOptions: UsePollingOptions) => {
 
     const cleanup = () => {
       if (pollTokenRef.current) {
+        removePollingTokenFromAppState(pollTokenRef.current);
```

Comment:
Do we ever want to stop tracking a token without stopping polling? Or the other way around - stop polling without removing a token from the app state? This seems like it should be one single function, rather than two.

Reply:
Ahh i see. Yeah they should be coupled in another function

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/23594/commits/3b4fb8f0355bc5f50430d25d6b1cfe5cbbfbca56

Reply:
apologies. done here instead https://github.com/MetaMask/metamask-extension/pull/23594/commits/029f53e1be6275aa9977f9d054854381dc3e981e

made more sense to couple in the actions